README.md

```diff
@@ -106,12 +106,12 @@ room_api = RoomApi(server_address='localhost:5002')
 # that allow to use HLS.
 address, room = room_api.create_room(video_codec="h264")
 
-# Create new room api with returned jellyfish address as a room could be
-# created on a different jellyfish instance
-# (if you communicate with a cluster of jellyfishes)
+# Create new room api with returned fishjam address as a room could be
+# created on a different fishjam instance
+# (if you communicate with a cluster of fishjames)
```

Comment:
We should decide whether the plural form is `fishjam` or `fishjames`.

Reply:
fishjams definitely